assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -64,6 +65,7 @@ function mapStateToProps(state) {
     accountHolderConfirmation: state.page.directDebit.accountHolderConfirmation,
     formError: state.page.directDebit.formError,
     phase: state.page.directDebit.phase,
+    countryGroupId: state.common.internationalisation.countryGroupId,
```

Comment:
While `mapStateToProps` is making `countryGroupId` available in the props of the `DirectDebitForm` component, it isn't being passed to the `LegalNotice` component. To make this happen you can add it on line 152 in this file:

```jsx
<LegalNotice countryGroupId={props.countryGroupId} />
```

Annoyingly this *should* have been flagged by the linter, but wasn't because we've used `/* eslint-disable react/no-unused-prop-types */` on line 34 to get around a bug in ESLint 🙈

Reply:
Done! Thanks.